### PR TITLE
new user login works.

### PR DESCRIPTION
**Pull request by dgreisen-cfpb**

### CODE REVIEW

File: devops/core/views.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ def sync(request):
 
 
 def home(request):
+    return HttpResponseRedirect('/static/dash.html')
```
Comment (m3brown) [inline devops/core/views.py]: Is this intentional of a leftover from debugging?

File: devops/core/views.py [hunk 2]
```diff
@@ -30,4 +31,20 @@ def home(request):
 
 def logout_view(request):
     logout(request)
+    return redirect('/static/dash.html')
```
Comment (m3brown) [inline devops/core/views.py]: Is this intentional of a leftover from debugging?

**Comment by m3brown** [comment]
Other than the comments above, :+1: from me for the backend code.

File: devops/core/views.py (hunk 1 above)
Comment (dgreisen-cfpb) [inline devops/core/views.py]: Right now, all the functionality is in the rich static app. That is not expected to be true forever. But until it is false, we are treating the static app as the home. so we just redirect home to it. It isn't elegant, but it works for now.

File: devops/core/views.py (hunk 2 above)
Comment (dgreisen-cfpb) [inline devops/core/views.py]: same as above

**Comment by dgreisen-cfpb** [comment]
The FE changes are what were discussed and worked on during the code review.

**Pull request merged**